lib/mayaUsd/ufe/ProxyShapeHierarchyHandler.h

```diff
@@ -16,7 +16,6 @@
 #pragma once
 
 #include <mayaUsd/base/api.h>
-#include <mayaUsd/ufe/UfeVersionCompat.h>
```

Comment:
This header is definitely needed for the UFE v1 case. I just added this last week. Also are you sure about some of the other UFE headers you've removed? They could be needed by some of the ifdef's.

Reply:
> This header is definitely needed for the UFE v1 case. 

I am on UFE 2.x. We should wrap such ifdef so it is clear. I would have never known if it is needed for UFE v1.

> Also are you sure about some of the other UFE headers you've removed? They could be needed by some of the ifdef's.

I am getting some help from a tool to detect these. It is not perfect but nonetheless very useful. We need to be be careful about blindly adding these headers since it can affect compile time and we do have a lot them.

Reply:
Normally I would agree, but this header it is all about UFE version compatibility so I don't think it should ever be conditionally included.